src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -480,6 +482,81 @@ public void ascii() throws IOException {
         );
     }
 
+    @Test
+    public void ifTest() throws IOException {
+        assertThat(
```

Comment:
split that to different tests or use parametrized tests

Reply:
ok will do